planner/core/testdata/plan_suite_out.json

```diff
@@ -350,9 +350,9 @@
       },
       {
         "SQL": "select /*+ USE_INDEX_MERGE(t1, c_d_e, f_g) */ * from t where c < 1 or f > 2",
-        "Best": "IndexMergeReader(PartialPlans->[Index(t.c_d_e)[[-inf,1)], Index(t.f)[(2,+inf]]], TablePlan->Table(t))",
```

Comment:
Expected, model2 prefers to use single-read plans(Table/IndexScan) instead of double-read plans(IndexLookup/IndexMerge/IndexJoin) to reduce the risks of large double-read requests.